pkg/testing/integration/program.go

```diff
@@ -206,6 +206,8 @@ type ProgramTestOptions struct {
 	DestroyOnCleanup bool
 	// Quick implies SkipPreview, SkipExportImport and SkipEmptyPreviewUpdate
 	Quick bool
+	// RequireService indicates that the test must be run against the Pulumi Service
+	RequireService bool
```

Comment:
Should we also copy this from overrides in the implementation of `With`, e.g.:

https://github.com/pulumi/pulumi/blob/3b843ffbfc8a344881fda0dfcb6881c9be20220a/pkg/testing/integration/program.go#L467-L469

(I think I may have forgotten to do that for some of the fields I've added to `ProgramTestOptions` in the past.)

Reply:
Good catch!